cdnsystem/config/path_linux.go

```diff
@@ -0,0 +1,23 @@
+/*
+ *     Copyright 2020 The Dragonfly Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+const (
+	DefaultDiskBaseDir = "/home/admin/ftp"
+
+	DefaultMemoryBaseDir = "/dev/shm/dragonfly"
```

Comment:
`Linux`and `Darwin` should use the same type of directory, if you use the tmp directory, please add the tmp directory in `internal/dfpath`.